asana/client.py

```diff
@@ -83,8 +83,16 @@ def request(self, method, path, **options):
                 response = getattr(self.session, method)(
                     url, auth=self.auth, **request_options)
                 if response.status_code in STATUS_MAP:
-                    raise STATUS_MAP[response.status_code](response)
-                elif response.status_code >= 500 and response.status_code < 600:
+                    # TODO: Change back to default behavior once 1.0 & 1.1 have
+                    # the same premium only response
+                    asana_error = STATUS_MAP[response.status_code](response)
+                    premium_only_str = "not available for free"
+                    if isinstance(asana_error, error.ForbiddenError) and (
+                            premium_only_str in str(asana_error)):
```

Comment:
The `AsanaError` class doesn't define a `__str__` magic method, so `str(asana_error)` is dubious. Prefer instead `asana_error.args[0]` which is a defined attribute inherited from Python's `Exception`. Alternatively, edit the base class `AsanaError` to save the message it builds as an instance attribute like `self.message`.

Reply:
Yeah I originally did `exception.msg` but it's not compatible with Python 3, and went off of the docs for `BaseException`. That being said for clarity, I'll just go with the route of storing the message as a part of the exception.